README.md

```diff
@@ -156,7 +158,8 @@ but this allow to add the same source in different sections to enrich using the
         ]
         "git": [
             "https:/github.com/chaoss/grimoirelab-perceval",
-            "https:/github.com/chaoss/grimoirelab-sirmordred"
+            "https:/github.com/chaoss/grimoirelab-sirmordred",
+            "https://username:api_token@github.com/chaoss/grimoirelab-private"
```

Comment:
this example can be left here tho (no explanation needed)

I would reduce the example
```suggestion
            "https://<username>:<api-token>@github.com/chaoss/grimoirelab-sirmordred"
```